tests/FunderMaps.IntegrationTests/Backend/Application/OrganizationUserAdminTests.cs

```diff
@@ -23,7 +23,7 @@ public OrganizationUserAdminTests(AuthBackendWebApplicationFactory factory)
         public async Task CreateOrganizationUserReturnOrganizationUser()
         {
```

Comment:
There is no test case for the password modification or the role modification. Same for the non-admin call. TODO or issue? Or append to PR?